tasks/page/processGerbers.js

```diff
@@ -11,17 +11,35 @@ const gerberFiles = require('../../src/gerber_files')
 if (require.main !== module) {
   module.exports = function(config, folder) {
     let file
-    if (fs.existsSync(`${folder}/kitnic.yaml`)) {
-      file = fs.readFileSync(`${folder}/kitnic.yaml`)
-    } else if (fs.existsSync(`${folder}/kitspace.yaml`)) {
-      file = fs.readFileSync(`${folder}/kitspace.yaml`)
-    } else if (fs.existsSync(`${folder}/kitspace.yml`)) {
-      file = fs.readFileSync(`${folder}/kitspace.yml`)
+    let projectPath
+    let repoRootPath = folder
+
+    repoFolders = folder.split('/')
+    if (repoFolders.length > 4) {
+      repoRootPath = repoFolders.slice(0, 4).join('/')
+      projectPath = repoFolders.splice(4).join('/')
+    }
+
+    if (fs.existsSync(`${repoRootPath}/kitnic.yaml`)) {
+      file = fs.readFileSync(`${repoRootPath}/kitnic.yaml`)
+    } else if (fs.existsSync(`${repoRootPath}/kitspace.yaml`)) {
+      file = fs.readFileSync(`${repoRootPath}/kitspace.yaml`)
+    } else if (fs.existsSync(`${repoRootPath}/kitspace.yml`)) {
+      file = fs.readFileSync(`${repoRootPath}/kitspace.yml`)
     }
-    const info = file == null ? {} : yaml.safeLoad(file)
+    let info = file == null ? {} : yaml.safeLoad(file)
     const files = globule
       .find(`${folder}/**/*`)
       .map(p => path.relative(folder, p))
```

Comment:
I think this should be searching `repoRootPath` and be relative to it. Maybe this solves your issue?

Reply:
Thanks @kasbah, that was the issue.